responsibleai/tests/test_exceptions.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+
+from responsibleai.exceptions import (ConfigAndResultMismatchException,
+                                      DuplicateManagerConfigException,
+                                      SchemaErrorException,
+                                      SystemErrorException,
+                                      UserConfigValidationException,
+                                      UserErrorException)
+
+TEST_MESSAGE = "Test message"
+
+
+class TestExceptions:
+    def test_user_exceptions(self):
+        ucve = UserConfigValidationException(TEST_MESSAGE)
+        assert ucve._error_code == 'Invalid config'
+        assert str(ucve) == TEST_MESSAGE
+        assert isinstance(ucve, UserErrorException)
+
+        de = DuplicateManagerConfigException(TEST_MESSAGE)
+        assert de._error_code == 'Duplicate RAI configuration detected.'
```

Comment:
nit: I think you could refactor this slightly by calling a method that does these asserts, and passing to the method the exception object, expected error code, the TEST_MESSAGE and the type of exception

Reply:
Done